legate/driver/launcher.py

```diff
@@ -260,6 +260,9 @@ def _compute_env(self) -> tuple[EnvDict, set[str]]:
         if system.env.get("PYTHONFAULTHANDLER", "") == "":
             env["REALM_BACKTRACE"] = "1"
 
+        if "CUTENSOR_LOG_LEVEL" not in system.env:
+            env["CUTENSOR_LOG_LEVEL"] = "1"
```

Comment:
could use `env.setdefault("CUTENSOR_LOG_LEVEL", "1")`

six / half-dozen

Reply:
I think `env` is not initialized from `system.env`, it starts out empty.

Reply:
:+1: I read too quickly